api-definitions.md

```diff
@@ -0,0 +1,215 @@
+# FluxManager
+## Flux
+```ts
+interface FluxModel {
+    uid: string, //uuid
+    name: string,
+    owner: string, //group
+    value: string, //json with configuration
+    resourceDependencies: string[], //resources paths,
+    lastEdited: Date //use a timestamp for http
```

Comment:
in et out dans la value ?

Reply:
Yep, dans l'idée tout ce que @Eliott-C-13 va définir comme besoin pour sa commande FFMPEG sera là, ça inclut le in et out